packages/app/src/components/Layout/Layout.js

```diff
@@ -1,78 +1,15 @@
 import React from 'react';
-import { connect } from 'react-redux';
+import DefaultLayout from './DefaultLayout/DefaultLayout';
 
-import Axis from './Axis';
-import { sGetUiLayout } from '../../reducers/ui';
-import { sGetDimensions } from '../../reducers/dimensions';
-import { colors } from '../../colors';
-
-const styles = {
-    container: {
-        display: 'grid',
-        gridTemplateColumns: '1fr 2fr',
-    },
-    item: {
-        borderColor: colors.greyLight,
-        borderStyle: 'solid',
-        borderWidth: '0 0 1px 1px',
-    },
-    filters: {
-        gridRowStart: 'span 2',
-    },
+const types = {
+    default: DefaultLayout,
 };
 
-const axisStyles = {
-    columns: {
-        height: 36,
-    },
-    rows: {
-        height: 36,
-    },
-    filters: {
-        height: 81,
-    },
+const getLayoutByType = (type, props) => {
+    const Layout = types[type];
+    return <Layout {...props} />;
 };
 
-class Layout extends React.Component {
-    state = {
-        dimensions: [],
-    };
-
-    renderAxisDropzone = axisName => {
-        const itemStyle = {
-            ...styles.item,
-            ...styles[axisName],
-        };
-
-        return (
-            <div
-                className={`${axisName}-container grid-item`}
-                style={itemStyle}
-            >
-                <Axis
-                    axisName={axisName}
-                    axisStyle={{ ...axisStyles[axisName] }}
-                />
-            </div>
-        );
-    };
-
-    render() {
-        return (
-            <div className="layout-container" style={styles.container}>
-                {this.renderAxisDropzone('columns')}
-                {this.renderAxisDropzone('filters')}
-                {this.renderAxisDropzone('rows')}
-            </div>
-        );
-    }
-}
-
-Layout.displayName = 'Layout';
-
-const mapStateToProps = state => ({
-    layout: sGetUiLayout(state),
-    dimensions: sGetDimensions(state),
-});
+const Layout = ({ type = 'default' }) => getLayoutByType(type);
```

Comment:
getLayoutByType takes `props` as the second argument, but a value is not provided.